README.md

```diff
@@ -77,7 +87,7 @@ fixes dependencies where bundled libraries depend on each other. If this option
 > If the output directory does not exist, create it.
 
 A command may look like
-`% dylibbundler -od -b -x ./HelloWorld.app/Contents/MacOS/helloworld -d ./HelloWorld.app/Contents/libs/`
+`% dylibbundler -cd -b -f -a ./HelloWorld.app -x ./HelloWorld.app/Contents/PlugIns/printsupport`
```

Comment:
Not sure about the use of "PlugIns/printsupport" here, this might be needlessly specific and confusing. What was wrong with the previous example?

Reply:
to show that the -x flag can be used to add additional executable files in an app bundle (besides the primary executable and its LC_LOAD_DYLIB dependencies. The -a flag provides an alternative way to fix-up app bundles. While most simple apps may only need the -a flag with the location of the app bundle, more complicated app bundles may require additional binary files (plugins) fixed up by adding them with -x flags. I've added back your (excellent) example. Thank you for creating this very useful tool.